tests/yast2_gui/yast2_firewall_set_default_zone.pm

```diff
@@ -0,0 +1,42 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: YaST2 Firewall UI test checks verious configurations and settings of firewall
+# Make sure yast2 firewall can opened properly. Configurations can be changed and written correctly.
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+use base "y2_module_guitest";
+use strict;
+use warnings;
+use testapi;
+use utils;
+use network_utils 'iface';
+use YaST::Module;
+
+sub run {
+    my $self = shift;
+    my $iface = iface();
+
+    $self->select_serial_terminal();
+    validate_script_output("firewall-cmd --get-default-zone", sub { m/public/ }, proceed_on_failure => 1);
+    select_console 'x11', await_console => 0;
+    YaST::Module::open(module => 'firewall', ui => 'qt');
+    $testapi::distri->get_firewall()->select_interfaces_page();
```

Comment:
from line 26 to line 34 we don't need any blank line, all in the same block, it is the core of the test, we could also apply to rest of the test modules.